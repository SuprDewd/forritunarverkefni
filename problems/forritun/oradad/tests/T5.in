-53 16 43 -94
