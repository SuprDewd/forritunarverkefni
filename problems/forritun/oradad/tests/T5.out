-53 -94 16 43
